lib/abide_dev_utils/xccdf.rb

```diff
@@ -10,14 +10,25 @@
 module AbideDevUtils
   # Contains modules and classes for working with XCCDF files
   module XCCDF
+    # Generate map for CEM
+    def self.gen_map(xccdf_file, opts)
+      type = opts.fetch(:type, 'cis')
+      case type.downcase
+      when 'cis'
+        new_map = Benchmark.new(xccdf_file).gen_map(**opts)
+      else
+        raise AbideDevUtils::Errors::ProfileUnsupportedError
+      end
+    end
+
     # Converts and xccdf file to a Hiera representation
     def self.to_hiera(xccdf_file, opts)
       type = opts.fetch(:type, 'cis')
       case type.downcase
       when 'cis'
         Benchmark.new(xccdf_file).to_hiera(**opts)
       else
-        AbideDevUtils::Output.simple("XCCDF type #{type} is unsupported!")
+        raise AbideDevUtils::Errors::UnsupportedXCCDFError
```

Comment:
Errors should have error messages. You can add an error message like this: `raise AbideDevUtils::Errors::UnsupportedXCCDFError, "Your error message here"`